packages/studio-base/src/PanelAPI/useConfigById.ts

```diff
@@ -2,14 +2,23 @@
 // License, v2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
+import * as _ from "lodash-es";
```

Comment:
Importing lodash as '_' is a common practice, it is even in their documentation.
But I agree on being selective about what to import, it can make difference when bundle the final build

Reply:
The thing I took the habit of importing directly the methods I needed but for some reason with lodash-es (not lodash) my editor imports it that way on its own

Reply:
Shouldn't it be done seperately then?

Reply:
Yes, it should. So we will address it internally later.